### ISSUE TITLE
Caracal 200dps

### ISSUE THREAD

**Issue by falkto** [pyfa-org/Pyfa#1527]
## Bug Report
Caracal
All 5
Rapid Light Missile Laincher II x5
Caldary Navy Nova Light Missile
Ballistic Control System II x4
210 dps!  Must be 330+
### Expected behavior:
330 dps

### Actual behavior:
220 dps

### Detailed steps to reproduce:
create new fit, add modules

### Fits involved in EFT format (Edit > To Clipboard > EFT):
[Caracal, гайко фриго убиватор]

Ballistic Control System II
Ballistic Control System II
Ballistic Control System II
Ballistic Control System II

[Empty Med slot]
[Empty Med slot]
[Empty Med slot]
[Empty Med slot]
[Empty Med slot]

Rapid Light Missile Launcher II, Caldari Navy Nova Light Missile
Rapid Light Missile Launcher II, Caldari Navy Nova Light Missile
Rapid Light Missile Launcher II, Caldari Navy Nova Light Missile
Rapid Light Missile Launcher II, Caldari Navy Nova Light Missile
Rapid Light Missile Launcher II, Caldari Navy Nova Light Missile

[Empty Rig slot]
[Empty Rig slot]
[Empty Rig slot]

### Operating system and version (eg: Windows 10, OS X 10.9, OS X 10.11, Ubuntu 16.10):
win 7.1 sp1
pyfa 1.36.0

**Comment by Neugeniko** [comment]
Hi,
     Thanks for your bug report.  In Window>Preferences>Fitting engine is 'Factor in reload time ...' selected? Turn it off.

Cheers,
Neug

**Comment by falkto** [comment]
yes, factor on, sorry for you time )) 24.04.2018, 14:31, "Neugeniko" <notifications@github.com>:Hi,Thanks for your bug report. In Window>Preferences>Fitting engine is 'Factor in reload time ...' selected? Turn it off.Cheers,Neug—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.
